packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@tanstack-query-firebase/react",
-	"version": "1.0.7",
+	"version": "2.0.8",
```

Comment:
is the bump to `2.x` intentional/necessary? Is there a big breaking change here?